WeakAurasOptions/OptionsFrames/MoverSizer.lua

```diff
@@ -49,6 +49,10 @@ local function moveOnePxl(direction)
     local physicalWidth, physicalHeight = GetPhysicalScreenSize();
     local oneX = GetScreenWidth() / physicalWidth
     local oneY = GetScreenHeight() / physicalHeight
+    if IsShiftKeyDown() then
```

Comment:
Problem with this feature is that when we press shift, it change moversizer alpha to zero, so we click on the void
Maybe that behavior needs to be changed when cursor is close to the buttons or something?

Reply:
Yeah, I tested the change in the last few weeks and I did find it annoying once already, so I think I'll drop the change.